ArduCopter/mode_auto.cpp

```diff
@@ -1991,12 +1991,12 @@ bool ModeAuto::verify_payload_place()
     case PayloadPlaceStateType_Ascending_Start: {
         Location target_loc(inertial_nav.get_position_neu_cm(), Location::AltFrame::ABOVE_ORIGIN);
         target_loc.alt = nav_payload_place.descend_start_altitude;
-        wp_start(target_loc);
+        auto_takeoff_start(target_loc.alt, target_loc.get_alt_frame() == Location::AltFrame::ABOVE_TERRAIN);
```

Comment:
we set the alt as fixed Location::AltFrame::ABOVE_ORIGIN so the  ABOVE_TERRAIN will be always false, is that in prevision of a following change ?

Reply:
I agree it's a bit pointless creating the target_loc local variable.  It would be better to instead directly change the three lines to this single line:

`auto_takeoff_start(inertial_nav.get_position_z_up_cm(), false);`

Reply:
Still looks unchanged to me..